operations/client.go

```diff
@@ -47,7 +47,7 @@ func printStatus() cli.Command {
 			opts := rest.ClientOptions{
 				Host:   c.Parent().String(clientHostFlag),
 				Port:   c.Parent().Int(clientPortFlag),
-				Prefix: "",
+				Prefix: "/rest",
```

Comment:
Should this be `rest/v1`? I think it would make more sense to put the `v1` here instead of in all the rest routes in `rest/client.go` but that would be an annoying change so feel free to ignore.

Reply:
On second thought, some routes may want to use a future v2 and others v1. This is fine.